package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eth-trezor-keyring",
-  "version": "0.5.1",
+  "version": "0.5.2",
```

Comment:
Nit: We usually handle the version bump in a separate PR. That can be useful if we ever need to revert a specific change, and so that we can have an independent discussion about the changes in a release.

Here's the process we use for libraries (it was written in a Google Doc somewhere too but I can't recall where): https://github.com/MetaMask/controllers#release--publishing

Reply:
It ends up looking less silly on repos that have a changelog. That's still a TODO here.